core/components/ui/carousel/carousel.makeswift.client.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
```

Comment:
Separate the Makeswift client component implementation from the component registration; the registration needs to happen both on the client and on the server.